cmd/compose/port.go

```diff
@@ -40,10 +41,15 @@ func portCommand(p *ProjectOptions, dockerCli command.Cli, backend api.Service)
 		ProjectOptions: p,
 	}
 	cmd := &cobra.Command{
-		Use:   "port [OPTIONS] SERVICE PRIVATE_PORT",
-		Short: "Print the public port for a port binding",
-		Args:  cobra.MinimumNArgs(2),
+		Use:   "port [OPTIONS] SERVICE [PRIVATE_PORT]",
+		Short: "List port mappings or a specific mapping for the service",
```

Comment:
```suggestion
		Short: "List port mappings or print the public port for a specific mapping for the service",
```

Reply:
this is just the same help used in docker/cli